### PR TITLE
Use black to format code

### PR DESCRIPTION
**Pull request by drolando**
Similar to what we're doing in other repos, let's use black to format the code to ensure a consistent style

### CODE REVIEW

File: tests/encoding/_decoders_test.py [hunk 1]
```diff
@@ -144,79 +134,81 @@ def test__decode_thrift_annotations_local_span(self, thrift_endpoint):
...
 
-        tags, local_endpoint, remote_endpoint = \
-            decoder._convert_from_thrift_binary_annotations(
-                thrift_binary_annotations,
-            )
+        (
+            tags,
+            local_endpoint,
+            remote_endpoint,
+        ) = decoder._convert_from_thrift_binary_annotations(thrift_binary_annotations,)
```
[the remark this answers is not in the document]
Comment (drolando) [reply tests/encoding/_decoders_test.py]: It doesn't always add them, which makes me think it's probably a bug. They're only there if previously the arguments were all on different lines and the last one had a trailing comma.

[the remark this answers is not in the document]
Comment (drolando) [reply tests/encoding/_decoders_test.py]: After replacing them with sed black doesn't put them back 🤷‍♂

File: py_zipkin/encoding/_decoders.py [hunk 2]
```diff
@@ -89,22 +86,18 @@ def _convert_from_thrift_endpoint(self, thrift_endpoint):
         """
         ipv4 = None
         ipv6 = None
-        port = struct.unpack('H', struct.pack('h', thrift_endpoint.port))[0]
+        port = struct.unpack("H", struct.pack("h", thrift_endpoint.port))[0]
```
Comment (adriancole) [inline py_zipkin/encoding/_decoders.py]: this reminds me of eslint in reverse

File: tests/encoding/_decoders_test.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (qudade) [reply tests/encoding/_decoders_test.py]: yes, it is a bug: https://github.com/psf/black/issues/1202. I think this will get resolved in the next version.

**Pull request merged**